### PR TITLE
Reminder Plugin

### PR DESCRIPTION
**Pull request by martinbndr**
Created a reminder plugin

### CODE REVIEW

File: plugins/registry.json [hunk 1]
```diff
@@ -34,6 +34,15 @@
         "title": "Suggest stuff.",
         "icon_url": "https://i.imgur.com/qtE7AH8.png",
         "thumbnail_url": "https://i.imgur.com/qtE7AH8.png"
+    },
+	"reminder": {
```
Comment (sebkuip) [inline plugins/registry.json:38]: Indentation error here. The name of the plugin should be indented only once (not twice)

Comment (martinbndr) [reply plugins/registry.json:38]: image
Looks fine for me on my end hmm

Comment (martinbndr) [reply plugins/registry.json:38]: Also tried a different program, its showing like this.
Do you have an idea wihy its wrong on github?
image

**Comment by StephenDaDev** [comment]
The plugin does not seem to install correctly onto a stock modmail bot. See logs: [https://hastebin.cc/yalopivare](https://hastebin.cc/yalopivare). While I could probably get it to work without erroring on me, plugins designed to be in the registry should work OOB, as most people using modmail don't really understand the technical side of things.

File: plugins/registry.json (hunk 1 above)
Comment (StephenDaDev) [reply plugins/registry.json:38]: Yeah it looks fine in VSC, (incl. the GH hosted VSC) not sure why it appears weird on the normal instance

**Comment by martinbndr** [comment]
image
Hmm strange, I tested installation without any plugins and it works

**Comment by martinbndr** [comment]
Update: now it should work, @StephenDaDev  Sry I made a stupid mistake

**Comment by lorenzo132** [comment]
Can confirm it now works

**Review by StephenDaDev — APPROVED** [review]
Tested as working, formatting is all good

**Review by Taaku18 — APPROVED** [review]

**Pull request merged**